nwb_conversion_tools/utils/spike_interface.py

```diff
@@ -750,6 +750,46 @@ def add_epochs(recording: RecordingExtractor, nwbfile=None, metadata: dict = Non
                 )
 
 
+def add_device_electrode_info(recording: RecordingExtractor, nwbfile=None, metadata: dict = None):
```

Comment:
```suggestion
def add_electrodes_info(recording: RecordingExtractor, nwbfile=None, metadata: dict = None):
```
Minor naming change; the device and electrode groups are both information fields of the electrodes, so I'd say the purpose of this function is just to add the ElectrodeTable and any of its dependencies.